tests/ui/writeln_empty_string.stderr

```diff
@@ -1,10 +1,16 @@
-error: using `writeln!(v, "")`
+error: using writeln!(&mut v, "")
```

Comment:
now I had to remove the ticks as I remove it from the code.
I didn't quite get the part with: `the backticks around writeln!() are automatically added by the span_lint_and_sugg function` since this is not the case right now.

Reply:
Oh, I meant they're added automatically to the `sugg` argument. Everything else you have to add them manually.